### ISSUE TITLE
Attributes of lost/gotpointercapture events fired immediately on set/release capture

### ISSUE THREAD

**Issue by mustaqahmed** [w3c/pointerevents#113]
This is blocked on #32. I am creating this new issue because @NavidZ's [comment](https://github.com/w3c/pointerevents/issues/32#issuecomment-232687047) there needs a bit more attention.

Since Edge is not delaying <code>lostpointercapture</code>/<code>gotpointercapture</code> in some cases and Chrome will be following a similar approach soon, the attribute values for, say, <code>button</code> or <code>pressure</code> seems to become irrelevant for these <code>lostpointercapture</code> and <code>gotpointercapture</code> events (i.e. for the ones fired immediately after a call to <code>setPointerCapture</code> or <code>releasePointerCapture</code>) because there is no hardware events to provide us the values here.

I see three possible ways to spec this:
A. Don't say anything, and expect UAs to pick up a consistent behavior (or expect developers rely on only the meaningful attributes).
B. These events set only the meaningful attributes (only <code>pointerId</code>?). All other attributes are either default or unknown.
C. These events always carry the attributes of the last known event for the <code>pointerId</code>, for both immediate firing and lazy firing in response to a real event.

Comments?

**Comment by RByers** [comment]
Discussed this at the hackathon.  We agree it probably doesn't really matter, but assuming we go with the delayed design (#32) then the simplest / least surprising thing to do is to just copy all the properties from the pointer event that triggered the sending of the lost/got event.  There may still be some cases where a `pointercancel` event has default values, and those get copied into the `lostpointercapture` but that's fine.

Thoughts?

**Comment by NavidZ** [comment]
Sounds good. Do you like to explicitly call that out about got/lostpointercapture in the spec that their attributes need to be the same as the PointerEvent that caused them to be processed? Note that currently Edge doesn't do that for lostpointercapture. Were the Edge folks okay with this to change lostpointercapture?

**Comment by RByers** [comment]
Yes, I think we should call this out in the spec and include testing for it.  Edge devs indicated it was a reasonable change for them to make.

**Comment by teddink** [comment]
I added this bug for Edge tracking: [https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/8315611/](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/8315611/)
